package.json

```diff
@@ -58,6 +58,7 @@
     "@pinia/nuxt": "^0.1.8",
     "@popperjs/core": "^2.11.2",
     "@tailwindcss/line-clamp": "^0.3.1",
+    "@tailwindcss/typography": "^0.5.2",
```

Comment:
I guess I don't fully understand why we're using this plugin and why if we _do_ use it it doesn't render the code in `tailwind.css` unnecessary (should we remove it if we use this plugin?).

Reply:
Makes sense, thanks!